packages/catalog-process/src/model/domain/errors.ts

```diff
@@ -39,11 +40,9 @@ export function eServiceNotFound(eserviceId: EServiceId): ApiError<ErrorCodes> {
   });
 }
 
-export function eServiceDuplicate(
-  eServiceNameSeed: string
-): ApiError<ErrorCodes> {
+export function eServiceDuplicate(eServiceName: string): ApiError<ErrorCodes> {
   return new ApiError({
-    detail: `ApiError during EService creation with name ${eServiceNameSeed}`,
+    detail: `EService with name: ${eServiceName} already in use`,
```

Comment:
What about this? `An EService with name ${eServiceName} already exists`

Reply:
Better! I copied the message as it is from the Scala codebase but I'd say we can improve it like that